CHANGELOG.md

```diff
@@ -1,3 +1,12 @@
+## 0.5.0
```

Comment:
Can please you omit the 0.5.0 additions to the `CHANGELOG`, but include them in a comment on this PR? I usually try to keep all `CHANGELOG` updates inside of the tagged commit of the release.

Keeping the 0.4.0 changes makes sense because they should have been there already.